backend/app.py

```diff
@@ -6,16 +6,16 @@
 app = Flask(__name__)
 CORS(app, supports_credentials=True)
 CORS_ALLOW_CREDENTIALS = True
-CORS_ORIGIN_WHITELIST = (
-    'http://localhost:5173',
-    'http://localhost:5000'
-	),
+CORS_ORIGIN_WHITELIST = (("http://localhost:5173", "http://localhost:5000"),)
```

Comment:
styling here shouldn't change between commits, probably someone forgot about black linter

Reply:
You're right, I forgot to use it. I'll push another commit with it soon.